x/participation/keeper/total_allocation_test.go

```diff
@@ -0,0 +1,47 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/stretchr/testify/require"
+
+	testkeeper "github.com/tendermint/spn/testutil/keeper"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/participation/types"
+)
+
+func createNDelegations(ctx sdk.Context, tk testkeeper.TestKeepers, addr string, n int) ([]stakingtypes.Delegation, sdk.Dec) {
```

Comment:
Could it be rather part of the `testkeeper` package, the same way we do for `Mint` in https://github.com/tendermint/spn/pull/619?

We could have a file `staking` in the package

This can be initial work for https://github.com/tendermint/spn/issues/403